glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -34,7 +34,7 @@ definitions:
     # but these special cases can be removed once the number of legacy clients
     # sufficiently dwindles, likely in 2020H2.
     pattern: "^[a-z][a-z0-9-]{0,29}$\
-      |^deletion_request$|^bookmarks_sync$|^history_sync$"
+      |^deletion_request$|^bookmarks_sync$|^history_sync$|^session_end$|^glean_.*$"
```

Comment:
@jklukas where does the `session_end` ping come from?

Reply:
https://github.com/MozillaReality/FirefoxReality/blob/87543c362a47810d149eb109b6a82e6b06c19aa4/app/pings.yaml#L7

Reply:
Hah, right :) I event reviewed that :(